core/src/main/java/com/datastax/oss/driver/api/core/servererrors/OverloadedException.java

```diff
@@ -34,8 +34,8 @@
  */
 public class OverloadedException extends QueryExecutionException {
 
-  public OverloadedException(@NonNull Node coordinator) {
-    super(coordinator, String.format("%s is bootstrapping", coordinator), null, false);
+  public OverloadedException(@NonNull Node coordinator, @NonNull String message) {
+    super(coordinator, message, null, false);
```

Comment:
I think we could do as in 3.x and augment the original message with the coordinator address:

```suggestion
    super(coordinator, String.format("%s was overloaded: %s", coordinator, message), null, false);
```

Reply:
Agree. Uploaded a new version of the commit, used "is" instead of "was" to be in-synch with the other error message (bootstraping). But let me know if you prefer was to be in synch with 3.x